state-manager-frontend/src/components/StatesByRunId.tsx

```
@@ -0,0 +1,331 @@
+            <button
+              onClick={() => setShowGraph(!showGraph)}
+              className="flex items-center space-x-2 px-4 py-2 bg-green-600 text-white rounded-lg hover:bg-green-700 transition-colors"
+            >
+              <Network className="w-4 h-4" />
+              <span>{showGraph ? 'Hide' : 'Show'} Graph</span>
+            </button>
+          )}
+          <button
+            onClick={loadCurrentStates}
+            className="flex items-center space-x-2 px-4 py-2 bg-[#031035] text-white rounded-lg hover:bg-[#0a1a4a] transition-colors"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)